examples/utils/example-utils/src/migrationTool/sameContainerMigrationTool.ts

```diff
@@ -316,16 +320,26 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
 		});
 
 		this._acceptedP = new Promise<void>((resolve) => {
-			if (this.pactMap.get(newVersionKey) !== undefined) {
-				console.log("Resolving this._acceptedP: Acceptance already exists at load time");
+			const pactWithDetails = this.pactMap.getWithDetails(newVersionKey);
+			if (pactWithDetails !== undefined) {
+				this._acceptedSeqNum = pactWithDetails.acceptedSequenceNumber;
+				console.log(
+					"Resolving this._acceptedP: Acceptance already exists at load time at sequence number:",
+					this._acceptedSeqNum,
+				);
 				resolve();
 				return;
 			}
 
 			const watchForAccepted = (key: string) => {
 				if (key === newVersionKey) {
 					this.pactMap.off("accepted", watchForAccepted);
-					console.log("Resolving this._acceptedP: Saw acceptance during run time");
+					this._acceptedSeqNum =
+						this.pactMap.getWithDetails(newVersionKey)?.acceptedSequenceNumber;
```

Comment:
Probably want to throw if undefined here (which should never be the case, but to clarify the expectation).  A non-null assertion could also be OK if you prefer -- but we're not OK with getWithDetails() returning `undefined`.